doc/docs/cli-installation.md

```diff
@@ -97,14 +97,14 @@ Install it via `pkg` from the [Ports Collection](https://www.freshports.org/deve
 $ pkg install coursier
 ```
 
-### zsh completions
+## zsh completions
```

Comment:
Previously it was a subsection of `JAR-based launcher` which was a bit misleading. That way it's IMHO clearer